airbyte-integrations/connectors/source-bing-ads/acceptance-test-config.yml

```diff
@@ -3,6 +3,8 @@ tests:
   spec:
     - spec_path: "source_bing_ads/spec.json"
   connection:
+    - config_path: "secrets/config_old.json"
```

Comment:
can this extra config_path be deleted?

Reply:
This is a needed part for testing the support of old input configurations, is there any valid reason we need to remove it?

Reply:
makes sense!